Mathlib/Algebra/Module/Submodule/Localization.lean

```diff
@@ -5,6 +5,8 @@ Authors: Andrew Yang
 -/
 import Mathlib.Algebra.Module.LocalizedModule
 import Mathlib.LinearAlgebra.Quotient.Basic
+import Mathlib.RingTheory.Flat.Basic
+import Mathlib.RingTheory.Localization.BaseChange
```

Comment:
I'm not sure if this is that good (a +50% increase).  
Can you move this into a new file `Mathlib.RingTheory.Flat.Localization`?

Reply:
Apparently imports are only changed in 4 files though.
I think probably flatness is the heavy import? Since flatness is such a basic notion, maybe we can split out a Flat/Defs file that excludes Injective/CharacterModule stuff.

Reply:
I think it is just because this file isn't used much for now. I think a new file is better as we will have more results on localization and flatness later.

Reply:
I see, will do!